origins_translations/config/install/origins_translations.languages.yml

```diff
@@ -1,18 +1,19 @@
-ar: ["Arabic", true, "ترجم هذه الصفحة", "اختر لغة"]
-zh-cn: ["Chinese (Simplified)", true,  "翻译此页", "选择语言"]
-zh-tw: ["Chinese (Traditional)", true, "翻譯此頁", "選擇語言"]
-nl: ["Dutch", true, "Vertaal deze pagina", "Selecteer een taal"]
-tl: ["Filipino", true, "Isalin ang pahinang ito", "Pumili ng isang wika"]
-fr: ["French", true, "Traduire cette page", "Sélectionnez une langue"]
-de: ["German", true, "Übersetze diese Seite", "Wähle eine Sprache"]
-hu: ["Hungarian", true, "Fordítsa le ezt az oldalt", "Válasszon nyelvet"]
-ga: ["Irish", true, "Aistrigh an leathanach seo", "Roghnaigh teanga"]
-it: ["Italian", true, "Traduci questa pagina", "Seleziona una lingua"]
-lv: ["Latvian", true, "Tulkojiet šo lapu", "Izvēlieties valodu"]
-lt: ["Lithuanian", true, "Išverskite šį puslapį", "Pasirinkite kalbą"]
-pl: ["Polish", true, "Przetłumacz tę stronę", "Wybierz język"]
-pt: ["Portuguese", true, "Traduza esta página", "Selecione um idioma"]
-ro: ["Romanian", true, "Traduceți această pagină", "Selectați o limbă"]
-ru: ["Russian", true, "Перевести эту страницу", "Выберите язык"]
-sk: ["Slovak", true, "Preložiť túto stránku", "Vyberte jazyk"]
-es: ["Spanish", true, "Traduzca esta pagina", "Selecciona un idioma"]
+ar: ["Arabic", "عربي", true, "ترجم هذه الصفحة", "اختر لغة", "rtl"]
```

Comment:
You'll need to add an update hook to update sites that already have the module installed.